source/uri-options/uri-options.rst

```diff
@@ -45,11 +45,15 @@ to the security implications of certain options, drivers MUST raise an
 error to the user during parsing if any of the following circumstances
 occur:
 
-1. Both ``tlsInsecure`` and ``tlsAllowInvalidCertificates`` appear in the 
+1. Both ``tlsInsecure`` and ``tlsAllowInvalidCertificates`` appear in the
    URI options.
-2. Both ``tlsInsecure`` and ``tlsAllowInvalidHostnames`` appear in the 
+2. Both ``tlsInsecure`` and ``tlsAllowInvalidHostnames`` appear in the
    URI options.
-3. All instances of ``tls`` and ``ssl`` in the URI options do not have the
+3. Both ``tlsInsecure`` and ``tlsDisableOcspEndpointCheck`` appear in
+   the URI options.
+4. Both ``tlsAllowInvalidCertificates`` and
```

Comment:
This doesn't seem right to me. tlsDisableOcspEndpointCheck doesn't disable verifying stapled OCSP responses, but tlsAllowInvalidCertificates does.